dwave_networkx/algorithms/tsp.py

```diff
@@ -133,6 +133,9 @@ def traveling_salesperson_qubo(G, lagrange=None, weight='weight'):
 
     weight : optional (default 'weight')
         The name of the edge attribute containing the weight.
+    
+    missing_edge_penalty : number, optional (default 'sum')
```

Comment:
This needs to be updated to match the function signature. Also should not be more than 80 characters wide.

Reply:
@arcondello, didn't we recently increase to 100 chars?

Reply:
The strict limit is 100, though in doc strings IMO it's nice to keep it under 80. It's easier in docstrings since it's all text, so wrapping rarely makes it less readable.